src/pudl/package_data/settings/etl_full.yml

```diff
@@ -215,6 +215,7 @@ datasets:
           2019,
           2020,
           2021,
+          2022,
         ]
       eia860m: true
```

Comment:
We need to disable EIA-860M here too don't we?

Reply:
I talked to @bendnorman about this and apparently not. But given that we don't have any eia860 monthly data right now we could probably set this to false anyways. It won't affect the ETL because we use the jobs defined in dagster of which only the fast tests is based on the yml file. I do think it might impact the tox tests however as those are the only files that sill use this etl_full.yml file I think.

Reply:
I think we should set it to `false` otherwise we're specifying settings which we expect to fail here.

If the builds aren't running based on these YML files, where do we need to update those settings?

Reply:
In the dagster launchpad for the etl_full job

Reply:
but I agree it should be false for the tox tests

Reply:
@bendnorman I'm definitely confused. Are the `etl_fast.yml` and `etl_full.yml` files not the source of the fast/full job configurations in Dagster?